operations/host_spawn.go

```diff
@@ -344,6 +362,103 @@ func hostStart() cli.Command {
 	}
 }
 
+func hostCreateVolume() cli.Command {
+	const (
+		zoneFlag = "zone"
+		sizeFlag = "size"
+		typeFlag = "type"
+	)
+
+	return cli.Command{
+		Name:  "create-volume",
+		Usage: "create a volume for spawn hosts",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  joinFlagNames(zoneFlag, "z"),
```

Comment:
If we aren't supporting availability zones in this ticket then I think we should simply remove the flag until we do.